modules/installation-osp-kuryr-ingress-scaling.adoc

```diff
@@ -52,44 +58,71 @@ spec:
   selector:
     ingresscontroller.operator.openshift.io/deployment-ingresscontroller: default
   sessionAffinity: None
-  type: LoadBalancer
+  type: LoadBalancer <2>
 ----
+<1> Ensure that this value is `LoadBalancer`.
```

Comment:
this is wrong. it should be ensure you change the name, such as router-external-default

Reply:
So `LoadBalancer` here is actually a placeholder, and you should change it to something descriptive, like `router-external-default`?

Reply:
exactly, that is the name you want to give you to the service (of LoadBalancer type) that you are creating